app/lib/denoms.ts

```diff
@@ -7,6 +7,9 @@ export const SUI = 9; // SUI -> mist decimals
 export const NBTC = 8; // NBTC decimals
 export const USDC = 6;
 
+// Auction constants
+export const MINIMUM_FIRST_BID_MIST = 1e9; // 1 SUI in mist for first-time bidders
```

Comment:
this should not be here. It's too specific. It should be either in the config (testnet / mainnet), or we can keep it as is in the BeelieversBid.